parkings/api/enforcement/valid_parking.py

```diff
@@ -29,9 +30,19 @@ class Meta:
             'is_disc_parking',
         ]
 
+    def __init__(self, *args, **kwargs):
+        super(ValidParkingSerializer, self).__init__(*args, **kwargs)
+        domain = self.context['request'].user.enforcer.enforced_domain
```

Comment:
Use the domain of the parking here, no the domain of the user executing the request. Although in valid_parking endpoint it doesn't matter so much, since they should be the same (since the enforcer executing the request should have access only to those parkings which are in their domain).

Reply:
I think this was not needed after all since the queryset filter already covered the domain check.

Reply:
I'm a bit uncertain if the slug field will work without the filtering if there are two zones with the same code, but we can add a test case for this in another PR.

Reply:
Ah, now I see why it works. However I think it might be better to go with a more explicit version, see #150.